configure.in

```diff
@@ -336,6 +336,12 @@ AC_CHECK_LIB(ssl, SSL_library_init, ,
 		AC_MSG_ERROR([libssl not found, but required])
 	]
 )
+LIBS="$LIBS -lcircllhist"
```

Comment:
I thought LIBS got updated automatically by AC_CHECK_LIB? so this line shouldn't be necessary?